saskatoon/harvest/api.py

```diff
@@ -1,15 +1,19 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.utils.translation import ugettext_lazy as _
 from django.urls import reverse, reverse_lazy
-from rest_framework import viewsets
 from rest_framework.response import Response
+from rest_framework import viewsets
 from django_filters import rest_framework as filters
-from .filters import ( HarvestFilter, PropertyFilter, EquipmentFilter,
-                       OrganizationFilter, CommunityFilter)
-from .forms import ( RequestForm, RFPManageForm, CommentForm, HarvestYieldForm )
+
+from harvest.filters import ( HarvestFilter, PropertyFilter, CommunityFilter,
+    OrganizationFilter, EquipmentFilter )
+from harvest.forms import ( RequestForm, CommentForm, RFPManageForm, 
+    HarvestYieldForm )
+
 from member.models import AuthUser, Organization
-from .models import Equipment, Harvest, HarvestYield, Property, RequestForParticipation, Comment
-from .serializers import ( HarvestSerializer, PropertySerializer, EquipmentSerializer,
+from harvest.models import ( Harvest, Property, HarvestYield, Comment,
+    Equipment, RequestForParticipation)
+from harvest.serializers import ( HarvestSerializer, PropertySerializer, EquipmentSerializer,
```

Comment:
@victorphoenix3 I'm sorry but I think you need to rebase your branch onto master. I've already pushed similar changes.

Reply:
Done